### ISSUE TITLE
Missing caption support to be WCAG compliant

### ISSUE THREAD

**Issue by joacim-boive** [t-obi/react-video-cover#220]
There's no way to add the TRACK-element so you can't add subtitles and thus you can't make this component WCAG compliant.

It could be as simple as render any children.
